src/workshops/4/conditions.c

```diff
@@ -0,0 +1,55 @@
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+enum { kTwo = 2, kFive = 5, kNine = 9 };
+
+/* Denne funktion skal returnere 1 hvis x er et primtal og 0 ellers */
+bool IsPrime(int number) {
+  for (int i = 2; i < number / 2; i++) {
+    if (number % i == 0) {
+      return false;
+    }
+  }
+  return true;
+}
+
+/* Denne funktion skal returnere 1 hvis gcd(x,2)=1 og 0 ellers */
+bool IsGcd1(int number);
+
+/* Denne funktion skal returnere 1 hvis 9^x-2 mod 5 = 2 og 0 ellers */
+bool Is2Mod5(int number) {
+  int exponent_calculation = (int)pow(kNine, number % kFive);
+  // NOLINTNEXTLINE(readability-implicit-bool-conversion)
+  return (exponent_calculation - kTwo) % kFive == kTwo;
+}
+
+int main(void) {
+  int number_to_test = 0;
+  int is_prime_number = 0;
```

Comment:
Why not use the `bool` type here as well?

Reply:
Do you know what the format specifier is for bool type? I can't seem to find it when googling.

Reply:
Idk, you can try to use `%d` or a ternary operator: https://www.programiz.com/c-programming/ternary-operator